sympy/functions/combinatorial/numbers.py

```diff
@@ -13,7 +13,7 @@
 from sympy.core import S, Symbol, Rational, oo, Integer, C, Add, Dummy
 from sympy.core.compatibility import as_int, SYMPY_INTS, xrange
 from sympy.core.cache import cacheit
-from sympy.core.numbers import pi
+from sympy.core.numbers import E, pi
```

Comment:
Try put it on the core.numbers import line?  Not sure how to expalin the test failure:

```
  File "/home/travis/virtualenv/python2.6.9/lib/python2.6/site-packages/sympy-0.7.6_git-py2.6.egg/sympy/functions/combinatorial/numbers.py", line 438, in _eval_rewrite_as_Sum
    return 1 / E * C.Sum(k**n / factorial(k), (k, 0, oo))
NameError: global name 'oo' is not defined
```

Also, we generally use S.Infinity in such cases rather than import oo. In that case all oo in the file could be changed to S.Infinity and the oo import removed.